index.html

```diff
@@ -407,17 +407,13 @@ <h3>I2C Multiplexers (Optional)</h3>
 
         <div id="components-section" class="section hidden">
             <h2>5. Add Components</h2>
-            <p style="color: red;"><small><i>UART + other component categories are currently being implemented in offline mode (No ETA, but maybe reactive actions too!).</i></small></p>
 
             <div id="component-type-tabs" class="tab">
                 <button class="comp-tab active" onclick="openComponentTab(event, 'all-components')">All Components</button>
                 <button class="comp-tab" onclick="openComponentTab(event, 'i2c-components')">I2C Components</button>
                 <button class="comp-tab" onclick="openComponentTab(event, 'ds18x20-components')">DS18x20 Components</button>
                 <button class="comp-tab" onclick="openComponentTab(event, 'pin-components')">Pin Components</button>
-                <button class="comp-tab not-available-yet" onclick="openComponentTab(event, 'pixel-components')">Pixel Components</button>
-                <button class="comp-tab not-available-yet" onclick="openComponentTab(event, 'pwm-components')">PWM Components</button>
-                <button class="comp-tab not-available-yet" onclick="openComponentTab(event, 'servo-components')">Servo Components</button>
```

Comment:
Good idea to hide these. These show up in all components, as far as I remember, so they could be hidden in the collection script.
We could also mention that output components are not supported in offline mode currently (like was here https://github.com/adafruit/Adafruit_Wippersnapper_Offline_Configurator/pull/5/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L410 )